bitnami/postgresql/templates/statefulset.yaml

```
@@ -562,6 +562,11 @@ spec:
   volumeClaimTemplates:
     - metadata:
         name: data
+        labels:
+          app: {{ template "postgresql.name" . }}
+          release: {{ .Release.Name }}
+          heritage: {{ .Release.Service }}
+          component: master        
```

We should use labels convention

```suggestion
        labels: {{- include "common.labels.standard" . | nindent 10 }}
          app.kubernetes.io/component: master    
```